ci/requirements-3.8-dev.yml

```diff
@@ -0,0 +1,57 @@
+channels:
+  - conda-forge
+dependencies:
+  - click
+  - clickhouse-cityhash
+  - clickhouse-driver>=0.0.8,<0.1.2
+  - clickhouse-sqlalchemy
+  - cmake
+  - flake8
+  - geoalchemy2
+  - geopandas
+  - google-cloud-bigquery>=1.0.0
+  - graphviz
+  - impyla>=0.15.0
+  - jinja2
+  - libiconv  # see https://github.com/jupyter/repo2docker/issues/758
+  - lz4
+  - multipledispatch>=0.6.0
+  - mypy
+  - numpy>=1.11
```

Comment:
you should bump numpy / pandas to the min for 3.8, I think 1.15 and 0.25.3; they don't really matter as that's what conda will install anyhow, but better to reflect in this file. also pyarrow >=0.15

Reply:
thanks @jreback for the review. I will work on these changes.

Reply:
I pinned pyarrow>=0.13 because pymapd has some restrictions `pyarrow >=0.12,<0.14`